targets/tizenrt-artik05x/apps/jerryscript/jerry_port.c

```diff
@@ -0,0 +1,121 @@
+/* Copyright JS Foundation and other contributors, http://js.foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <stdio.h>
+#include <string.h>
+#include <stdlib.h>
+#include <setjmp.h>
+#include <stdarg.h>
+
+#include "jerryscript.h"
+#include "jmem.h"
+
+
+/**
+ * JerryScript log level
+ */
+static jerry_log_level_t jerry_log_level = JERRY_LOG_LEVEL_ERROR;
```

Comment:
Before #1777 landed, jerry_port.c was linked in jerry library so application cannot access the jerry_log_level, defined in jerry library, without external API or making jerry_log_level public. But since for now, at least tizenrt port can follow your suggestion. For your information, currently `jerryscript-port-default` defines `void jerry_port_default_set_log_level (jerry_log_level_t level);`